packages/bundler-plugin-core/src/options-mapping.ts

```diff
@@ -52,57 +57,84 @@ export type InternalIncludeEntry = RequiredInternalIncludeEntry &
   };
 
 export function normalizeUserOptions(userOptions: UserOptions): InternalOptions {
-  let entries: (string | RegExp)[] | ((filePath: string) => boolean) | undefined;
-  if (userOptions.entries === undefined) {
-    entries = undefined;
-  } else if (typeof userOptions.entries === "function" || Array.isArray(userOptions.entries)) {
-    entries = userOptions.entries;
+  const entries = normalizeEntries(userOptions.entries);
+  const include = normalizeInclude(userOptions);
```

Comment:
L: When I first saw this, my instinct was to ask if there was reason for passing just `entries` to `normalizeEntries` but passing the full `userOptions` object to `normalizeInclude`. Having dug into the code as far as `normalizeIncludeEntry`, I can now see why, but might be worth a comment.

L: Also, is there a reason to do these separately, rather than inlining them in the return value, now that they're just function calls?

Reply:
[52564d9](https://github.com/getsentry/sentry-javascript-bundler-plugins/pull/103/commits/52564d965589053396952d23d85460923a2087d1)